Backtesting/collectors/exchanges/exchange_live_collector.py

```diff
@@ -28,24 +28,29 @@
     logging.error("ExchangeLiveDataCollector requires OctoBot-Trading package installed")
 
 
-class ExchangeLiveDataCollector(ExchangeDataCollector):
+class ExchangeLiveDataCollector(AbstractExchangeLiveCollector):
     IMPORTER = GenericExchangeDataImporter
 
     async def start(self):
         exchange_factory = create_new_exchange(self.config, self.exchange_name, is_simulated=True, is_rest_only=True,
-                                               is_collecting=True)
+                                               ignore_config=True, is_collecting=True)
         await exchange_factory.create_basic()
+        self._load_timeframes_if_necessary()
 
+        # create description
+        await self._create_description()
```

Comment:
If this is common to each start(), can we put `await self._create_description()` to the `AbstractExchangeLiveCollector.start()` or maybe to the  `ExchangeDataCollector.start()` ?